### ISSUE TITLE
[Proposal] Interactive flow-based scenario/agent configuration

### ISSUE THREAD

**Issue by ashleysommer** [cantino/huginn#1952]
A colleague showed me this website today: https://nodered.org/
I watched the video and was blown away the interactive node flow building tool.
Is it possible to build a similar feature for huginn?
I understand it would be a mammoth effort to implement, but I personally think it would reduce the learning curve and improve the usability Huginn, and definitely increase usage and uptake.

**Comment by cantino** [comment]
Yes, Node-RED has a very slick editor. If someone wants to tackle this, I'm not opposed, but it's not something I'm going to take on at the moment.

**Comment by alias1** [comment]
Related, but never really went anywhere with it: #1296
